### PR TITLE
Fix(eos_cli_config_gen): Merge flow_tracking tables

### PR DESCRIPTION
**Pull request by chetryan**
## Change Summary

Merge the new and deprecated tables for flow_tracking so that it's easy to understand the options.

## Related Issue(s)

NO issue raised

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

- Merge `flow-trackings.md` and `flow-tracking.md` files
- Update the Input Vars Markdown to refer only to `flow-tracking.md`

## How to test

- Refer to the documentation generated by the CI on this PR

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md [hunk 1]
```diff
@@ -611,12 +611,6 @@ roles/eos_cli_config_gen/docs/tables/event-monitor.md
 roles/eos_cli_config_gen/docs/tables/flow-tracking.md
 --8<--
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md:613]:
```suggestion

### Flow trackings

--8<--
roles/eos_cli_config_gen/docs/tables/flow-tracking.md
--8<--

```

**Comment by gmuloc** [comment]
For now there are still two tables in the rendered doc is it expected?

Screenshot 2023-12-04 at 14 29 14

**Comment by ClausHolbechArista** [comment]
@gmuloc My bad, the git diff did not make it easy to figure out what had been removed/added so I readded the old section. I think it is correct now.

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Review by gmuloc — APPROVED** [review]

**Pull request merged**